syntax = "proto3";
option cc_enable_arenas = true;

package Fq.Quota;

import "google/protobuf/timestamp.proto";

////////////////////////////////////////////////////////////

message EvQuotaUpdateNotification {
    string subject_type = 1;
    string subject_id = 2;
    string metric_name = 3;
    uint64 metric_limit = 4;
    google.protobuf.Timestamp limit_updated_at = 5;
    uint64 metric_usage = 6;
    google.protobuf.Timestamp usage_updated_at = 7;
}
